Add EKS Anywhere blog post (#1697)
pulumi/pulumi-hugo: themes/default/content/blog/amazon-eks-anywhere-bare-metal/index.md

---
title: Deploying Amazon EKS Anywhere on Bare Metal
date: 2022-07-01T13:03:41-07:00

> meta_desc: "AWS has now announced the latest innovation for EKS: Amazon EKS Anywhere Bare Metal."

Reviewer: ```suggestion
meta_desc: "Bring EKS to Your Datacenter with Amazon EKS Anywhere Bare Metal."
```

meta_image: meta.png
authors:
    - david-flanagan

# At least one tag is required. Lowercase, hyphen-delimited is recommended.
tags:

>     - change-me

Reviewer: ```suggestion
    - Kubernetes, EKS, AWS
```
